src/main/java/net/ccbluex/liquidbounce/ui/client/hud/element/elements/Arraylist.kt

```diff
@@ -112,12 +114,24 @@ class Arraylist(x: Double = 1.0, y: Double = 2.0, scale: Float = 1F,
     }
 
     private fun getDisplayString(module: Module): String {
+        val moduleName = when (moduleCase) {
+          "Uppercase" -> module.getName().uppercase()
+          "Lowercase" -> module.getName().lowercase()
+          else -> module.getName()
+        }
+
+        val moduleTag = when (tagsCase) {
+          "Uppercase" -> module.tag!!.uppercase()
+          "Lowercase" -> module.tag!!.lowercase()
+          else -> module.tag
+        }
+
         val displayString = when {
-            tags && !module.tag.isNullOrEmpty() -> module.getName() + tagPrefix + module.tag + tagSuffix
-            else -> module.getName()
+            tags && !module.tag.isNullOrEmpty() -> moduleName + tagPrefix + moduleTag + tagSuffix
```

Comment:
Move moduleTag evaluation here so that module.tag is null-safe.

Reply:
> Move moduleTag evaluation here so that module.tag is null-safe.

@CzechHek But how?